tendrl/ceph_integration/objects/pool/atoms/update/__init__.py

```diff
@@ -14,6 +16,20 @@ def run(self):
         attrs = {}
         if 'Pool.poolname' in self.parameters:
             attrs['name'] = self.parameters.get('Pool.poolname')
+            Event(
```

Comment:
I don't think we need to log messages on parameter by parameter basis. We can have a single message for the update and may be specify the params there